test/templates/realtime.tpl.java

```diff
@@ -0,0 +1,33 @@
+import io.kuzzle.sdk.*;
+import java.util.Date;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+public class CodeExampleGenericClass {
+    private static Kuzzle kuzzle;
+
+    public static void main(String[] args) {
+      kuzzle = new Kuzzle("kuzzle");
+      kuzzle.connect();
+
+      ByteArrayOutputStream baos = new ByteArrayOutputStream();
+      PrintStream ps = new PrintStream(baos);
+      PrintStream outOriginal = System.out;
+
+      System.setOut(ps);
+
+      [snippet-code]
+
+      while (baos.size() == 0) {
```

Comment:
You need to break from the loop after a maximum number of retries, to prevent infinite loops if the test fails